README.md

```diff
@@ -80,29 +49,8 @@ If you find a security issue with our libraries or services please report it to
 
 ## Contributing
 
-All code is licensed under the MIT license and we triage actively on GitHub. We enthusiastically welcome contributions and feedback. You can clone the repo and start contributing now.
+All code is licensed under the MIT license and we triage actively on GitHub. We enthusiastically welcome contributions and feedback. Please read the [contributing guide](./contributing.md) before starting.
 
 ## We Value and Adhere to the Microsoft Open Source Code of Conduct
 
 This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/). For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.
-
-## Quick Start
-
-### Installation
-
-``` $ pip install adal ```
-
-### http tracing/proxy
-If you need to bypass self-signed certificates, turn on the environment variable of `ADAL_PYTHON_SSL_NO_VERIFY`
```

Comment:
AFAIK, this particular line was the only place we document this behavior. If we are going to remove it here, do we plan to document it somewhere in the wiki then?

PS: We did mention it recently in [API reference doc](http://adal-python.readthedocs.io/en/latest/#adal.AuthenticationContext.__init__). But I assume DevOps will not necessarily read API Reference doc. I could be wrong, though.

Reply:
This content is moved to the [Basics page](https://github.com/AzureAD/azure-activedirectory-library-for-python/wiki/ADAL-basics#http-tracingproxy) of the Wiki.